pythran/tables.py

```diff
@@ -187,7 +188,7 @@ def update_effects(self, node):
         "clear": MethodIntr(),
         "copy": ConstMethodIntr(),
         "get": ConstMethodIntr(),
-        "has_key": ConstMethodIntr(),
+        "has_key": ConstMethodIntr(return_range=prange.bool_values),
```

Comment:
Hum, you are right. It could avoid the the decltype(...) in typing.
I will add a BR to not forget it.